wsmaster/che-core-api-workspace-shared/src/main/java/org/eclipse/che/api/workspace/shared/Constants.java

```diff
@@ -43,6 +43,8 @@
   public static final String WS_AGENT_PROCESS_NAME = "CheWsAgent";
 
   public static final String CHE_WORKSPACE_AUTO_START = "che.workspace.auto_start";
+  public static final String CHE_WORKSPACE_PLUGIN_REGISTRY_ULR =
```

Comment:
Please add few words to describe in which way this property supposed to be used (exposed by settings method).